acestep/api_server.py

```
@@ -540,6 +540,297 @@ class Config:
+    format_lyrics: bool = Field(default=False, description="Format user lyrics via LLM")
+    transcribe_lyrics: bool = Field(default=False, description="Transcribe lyrics from audio")
+    only_unlabeled: bool = Field(default=False, description="Only label unlabeled samples")
+
+    lm_model_path: Optional[str] = Field(default=None, description="Optional LM model path to use for labeling (temporary switch)")
+
+    save_path: Optional[str] = Field(default=None, description="Optional dataset JSON path to persist progress during auto-label")
+
+    chunk_size: int = Field(default=16, ge=1, description="Chunk size for batch audio encoding")
+    batch_size: int = Field(default=1, ge=1, description="Batch size for batch audio encoding")
+
+    @root_validator(pre=True)
```

Ruff (0.15.0)

[error] Undefined name `root_validator`

(F821)